Adeolu_form.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+ 
+    <form>
+<b><label>First_Name:</label>  </b>
+<input id="first name" 
+placeholder="Put your name here" 
+type="text"
+maxlength="7" 
+size="1">
```

Comment:
You should have a size larger than 1. If you run the page in your browser you will see that the textbox is too small